content/applications/hr/payroll.rst

```diff
@@ -457,22 +629,20 @@ advantage listed for a particular structure type is only available for that spec
    have meal vouchers. The opposite is true for someone hired under the :guilabel:`Intern` structure
    type. They would have meal vouchers available to them, not the use of a company car.
 
-To make a new advantage, click the :guilabel:`New` button, and enter the information in the fields.
-The required fields for an advantage are:
+To make a new benefit, click the :guilabel:`New` button, and enter the information in the fields.
+The required fields for a benefit are:
 
-- :guilabel:`Name`: Enter the name for the advantage.
-- :guilabel:`Advantage Field`: Select from the drop-down menu what field in the payslip this
-  advantage appears under.
-- :guilabel:`Advantage Type`: Select from the drop-down menu what type of advantage the benefit is.
-  Select from :guilabel:`Monthly Benefit in Kind`, :guilabel:`Monthly Advantages in Net`,
-  :guilabel:`Monthly Advantages in Cash`, or :guilabel:`Yearly Advantages in Cash`.
-- :guilabel:`Salary Structure Type`: Select from the drop-down menu which salary structure type this
-  advantage applies to.
-- :guilabel:`Display Type`: Select from the drop-down menu how this advantage is displayed.
+- :guilabel:`Name`: enter the name for the benefit.
+- :guilabel:`Related Type`: select from the drop-down menu what type of benefit it is. Select from
+  :guilabel:`Monthly Benefit in Kind`, :guilabel:`Monthly Benefits in Net`, :guilabel:`Monthly
+  Benefits in Cash`, :guilabel:`Yearly Benefits in Cash`, or :guilabel:`Non Financial Benefits`.
+- :guilabel:`Salary Structure Type`: select from the drop-down menu which salary structure type this
+  benefit applies to.
+- :guilabel:`Display Type`: select from the drop-down menu how this benefit is displayed.
```

Comment:
again, i feel like a lot of fields are being overlooked on this form. If there's a reason why, perhaps it would be best for the reader to know...?

Reply:
Agreed- i was only doing required fields. All updated!